src/js/media-controller.js

```diff
@@ -640,6 +644,10 @@ class MediaController extends MediaContainer {
   }
 
   propagateMediaState(stateName, state) {
+    if (arguments.length === 1) {
+      state = Delegates[stateName](this);
+    }
```

Comment:
Now that we have the Delegates concept I _think_ we could lean on those more in other areas, but I didn't want to do a big refactor in the middle of this one. I did however add this nicety to let propagateMediaState default to using the delegate, but interested if that could go wrong somehow.